plugins/modules/docker_swarm.py

```diff
@@ -507,13 +522,16 @@ def init_swarm(self):
                 'advertise_addr': self.parameters.advertise_addr,
                 'listen_addr': self.parameters.listen_addr,
                 'data_path_addr': self.parameters.data_path_addr,
+                'data_path_port': self.parameters.data_path_port,
```

Comment:
This line needs to be removed.

Reply:
You're absolutely right, forgot to take it off.